src/styles/_climate-setting-schedule-table.scss

```diff
@@ -0,0 +1,104 @@
+@use './colors';
+
+@mixin all {
+  .seam-climate-setting-schedule-table {
```

Comment:
copied from the scss file for access codes. 

Is it fine to duplicate like this? I noticed we nest all our classes, so I think that I have to copy this over instead of reusing.

Reply:
cc @mikewuu

Reply:
> Would it be preferable to have a common file with some of these classes? ex. seam-dot-divider.

Yep. Definitely in this case, looks like the styles are exactly the same (other than a couple class names).

- Create one-off classes to start
- If you find yourself copying/pasting styles to another class, refactor out to common class/file

In this case maybe we add a `seam-table`?

Reply:
Okay I saw that Supported devices table was copied over so I thought it was a convention here? I can start on a refactor.